app/views/article/_article_links.html.slim

```diff
@@ -1,11 +1,12 @@
 -unless @article.show_links(@collection).empty?
-  h5 ="#{pluralize(@article.show_links(@collection).size, 'page')} refer to #{@article.title}"
+  h5 ="#{pluralize((@article.show_links(@collection).count).count, 'page')} refer to #{@article.title}"
```

Comment:
I don't understand the second `.count` here.  Won't that blow up?

Reply:
I know, it looks totally bizarre.  Article.show_links now returns an active record relation on the page_article_links that's grouped by text_type and page number (so that it doesn't matter how many times it's on the page).  The first count gives you a hash of the the grouped relation, with the group as keys and the count as values (i.e., {["transcription", 26895]=>2, ["translation", 26895]=>1, ["transcription", 26899]=>1}).  And the second count gives you a count of the key/value pairs in the hash.  So it's weird because of the grouping, but it does use both counts.  Although I just double-checked, and we could use size instead of the second count, which might be a better choice for consistency with the other counts we have in the UI.